styles.css

```diff
@@ -455,3 +460,220 @@ fieldset {
   border-radius: 10px;
   margin: auto 27%;
 }
+
+@media (min-width: 768px) {
+  .hamberger-btn {
+    display: none;
+  }
+
+  .nav-link {
+    display: flex;
+    flex-direction: row;
+    color: #fff;
+    gap: 3%;
+  }
+
+  .navbtn {
+    padding: 5%;
+  }
+
+  nav {
+    margin-right: 11%;
+  }
+
+  .inbox {
+    width: 50%;
+    height: 50%;
+    margin-top: 5%;
+  }
+
+  .banner-section {
+    background-image: url(img/Illustration-header@2x.png);
+    background-size: 40%;
+  }
+
+  .container {
+    width: 327px;
+    gap: 12px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    margin-left: 12%;
+  }
+
+  .socials {
+    position: absolute;
+    left: 7%;
+  }
+
+  .iconsSection {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .recent-works {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-content: center;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .works-break {
+    width: 68%;
+    margin-right: 3%;
+  }
+
+  #works-title {
+    margin-left: 4%;
+  }
+
+  #grid-container {
+    display: grid;
+  }
+
+  .project-img {
+    width: 67%;
+  }
+
+  .recent-project {
+    display: flex;
+    flex-direction: row;
+    margin-left: 11%;
+    margin-right: 11%;
+  }
+
+  .right-block {
+    height: fit-content;
+    width: min-content;
+    gap: 27px;
+  }
+
+  .cards-box {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    gap: 10px;
+  }
+
+  .posts-description {
+    width: 173%;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #091e42;
+  }
+
+  .cardworks,
+  .card-btn {
+    display: none;
+  }
+
+  .card-works:hover,
+  .card-btn {
+    display: block;
+    height: 48px;
+    background: #ff6b00;
+    color: #fff;
+    border: none;
+  }
+
+  .card-works:hover h2,
+  .card-works:hover p {
+    color: transparent;
+  }
+
+  .card-works:hover ul {
+    display: none !important;
+  }
+
+  .about {
+    border-radius: 0;
+    padding: 130px 0 130px 0;
+    background-image: url(img/Frame1.png);
+    background-position-y: 14%;
+    background-size: 48%;
+  }
+
+  .about-me {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 130px 0;
+    gap: 24px;
+    width: 44%;
+    margin-left: 7%;
+  }
+
+  .about-breaker {
+    margin: 5% 57% 5% 6%;
+    width: 87%;
+  }
+
+  .tools-list {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    gap: 30%;
+    margin-right: 34%;
+    margin-top: 11%;
+  }
+
+  .listtopography {
+    width: fit-content;
+  }
+
+  .mini-list {
+    width: max-content;
+    display: flex;
+    align-items: flex-start;
+  }
+
+  .bottom-footer {
+    display: flex;
+    flex-direction: row;
+    gap: 0;
+  }
+
+  .contact-section {
+    height: auto;
+    width: 445px;
+  }
```

Comment:
- [ ] As mentioned previously please make sure your contact form looks similar to the given Figma design.
    - Your design:
      ![Inkedfirefox_NpRxUyp93R](https://github.com/Obimbo07/Portfolio-mobileFirst/assets/104968236/a3f93a2c-305a-4cc3-8f58-4ad416b1f7b6)
    - Figma design:
      ![firefox_NpRxUyp93R](https://github.com/Obimbo07/Portfolio-mobileFirst/assets/104968236/7ee692c8-2508-4f32-a466-f587fc4c7cbe)
- [ ] As you can see from your design you seem to be missing some spacing to the left of your text, you can fix that by adding a `margin-left: 6rem;` 😉